infrastructure/labelmaker/labeltester.jinja

```diff
@@ -0,0 +1,90 @@
+{#
+Copyright 2017 Google Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#}
+
+{% set COMPUTE_URL_BASE = 'https://www.googleapis.com/compute/v1/' %}
+{% set ID = env['deployment'] + '-' + env['name'] %}
+
+resources:
+- name: {{ ID }}
+  type: compute.v1.instance
+  properties:
+    zone: {{ properties['zone'] }}
+    machineType: {{ properties["machineType"] }}
+    metadata:
+      items:
+        - key: startup-script
+          value: |
+            #!/bin/bash -e
+            apt-get update
+            apt-get -y install python-pip
+            pip install --upgrade google-api-python-client
```

Comment:
pretty minor but you are relying on the "requests" package to be installed with python-pip.  It would be better if you explicitly listed it as a required package in the pip install command: 

pip install --upgrade google-api-python-client requests

to make it clear what the dependencies are. 
thanks